internal/pkg/warewulfd/util.go

```diff
@@ -34,24 +35,23 @@ func sendFile(
 		req,
 		filename,
 		stat.ModTime(),
-		fd )
+		fd)
 
 	wwlog.Send("%15s: %s", sendto, filename)
 
 	return nil
 }
 
 func getOverlayFile(
-	nodeId string,
+	n node.NodeInfo,
```

Comment:
I'd really rather this be something other than `n`. It takes me so much longer to understand what some of the Warewulf code is doing because of all these single-character variables. (I've tended to use `nodeInfo` in similar circumstances.)

Reply:
right, short names can be confusing.